src/utils/iframe.js

```diff
@@ -74,9 +74,9 @@ export function whenIframeLoaded(iframe) {
  *  - `writer` to be used for further updates.
  *    Takes `(iframe, srcdoc)` and returns `srcdoc` for chaining.
  */
-export const setSrcdocAsyncMultiStrategy = (win, iframeReady, srcdoc) =>
+export function setSrcdocAsyncMultiStrategy(win, iframeReady, srcdoc) {
   // https://caniuse.com/#search=srcdoc
-  'srcdoc' in win.HTMLIFrameElement.prototype
+  return false && 'srcdoc' in win.HTMLIFrameElement.prototype
```

Comment:
Let's remove the returned value for `false` and simplify:

1. `srcdoc` is no longer needed. Remove from templates.
2. `writer` is now constant, so replace `this.writeToIframe_` in `AmpStoryAdPreview` to `writeToIframe` calls as exported here.
3. Instead of using `iframeReady`, let's just write on construction on `AmpStoryAdPreview`:

```js
export default class AmpStoryAdPreview {
  constructor(win, element) {
    /** @private @const {!Window>} */
    this.win = win;

    const srcdoc = getDataTemplate(element).replace(
      "{{ adSandbox }}",
      defaultIframeSandbox
    );

    /** @private @const {!Promise<HTMLIFrameElement>} */
    this.storyIframe_ = untilAttached(element, s(".iframe"))
      .then(whenIframeLoaded)
      .then(iframe => {
        writeToIframe(iframe, srcdoc);
        return whenIframeLoaded(iframe);
      });

    // ...

    render(WrappedIframe(), element);
  }

  // ...
}
```